src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -490,6 +597,16 @@ public Curve AddCurveForColumn(DataColumn dataColumn, CurveOptions defaultCurveO
          return curve;
       }
 
+      private (bool exists, Curve curve) createAndConfigureCurve(DataColumn dataColumn, CurveOptions defaultCurveOptions)
+      {
+         var curve = Chart.CreateCurve(dataColumn.BaseGrid, dataColumn, _curveNameDefinition(dataColumn), _dimensionFactory);
+
+         if (Chart.HasCurve(curve.Id))
```

Comment:
How can this be false here? You just create the curve above?
Also returning named parameter (exists:true, etc..) is easier to read

Reply:
yeap true. just using Chart.FindCurveWithSameDataI() in case thw Curve already exists. Named the exist parameter too.